static/js/containers/App_test.js

```diff
@@ -43,6 +54,65 @@ describe('App', () => {
     });
   });
 
+  describe('dashboard errors', () => {
+    let errorString = `Sorry, we were unable to load the data necessary
+      to process your request. Please reload the page.`;
+    errorString = errorString.replace(/\s\s+/g, ' ');
+
+    it('error from the backend triggers error message in dashboard', done => {
```

Comment:
We just merged a PR which changes how these tests are written. I found that the test runner mocha natively handles tests with promises so instead of using `done` we can just return the promise in this function. For each test below can you remove `done` from the arguments, the `done()` call and use `return renderComponent(...` instead?